cmd/monaco/integrationtest/assert.go

```diff
@@ -318,14 +319,21 @@ func AssertBucket(t *testing.T, client client.BucketClient, env manifest.Environ
 	return expectedId
 }
 
-func getBucketWithRetry(ctx context.Context, client client.BucketClient, bucketName string, try, maxTries int) (buckets.Response, error) {
-	resp, err := client.Get(ctx, bucketName)
-	if try < maxTries && resp.StatusCode == http.StatusNotFound {
-		try++
+func getBucketWithRetry(ctx context.Context, client client.BucketClient, bucketName string, maxTries int) (buckets.Response, error) {
```

Comment:
So it changes it from 5 seconds max retry to 120 right?

Reply:
yes, correct - i suspect 5 is also not enough